CommunityFridgeMapApi/functions/image/v1/app.py

```diff
@@ -0,0 +1,45 @@
+import json
+import base64
+
+try:
+    from storage import Storage
+except:
+    from dependencies.python.storage import Storage
+
+class ImageHandler:
+    @staticmethod
+    def get_binary_body_from_event(event: dict) -> bytes:
+        """Extract binary data from request body"""
+        assert event["isBase64Encoded"]
+        return base64.b64decode(event["body"])
+
+    @staticmethod
+    def encode_binary_file_for_response(blob: bytes) -> bytes:
+        """
+        Binary response body of lambda functions should be encoded in base64.
+        https://docs.aws.amazon.com/apigateway/latest/developerguide/lambda-proxy-binary-media.html
+        """
+        return base64.b64encode(blob)
+
+    @staticmethod
+    def lambda_handler(event: dict, storage: Storage) -> dict:
+        bucket = "fridge-report"
+        key = storage.write(bucket, "webp", ImageHandler.get_binary_body_from_event(event))
```

Comment:
I did not validate the uploaded data here. Do we want to check if the uploaded data is a valid webp image?

Reply:
Yes we should check if it's valid webp, return an error code and message if it's not

Reply:
Added validation here f009311aaff1f17a5aa317aef2fb18e24c2f4a2a.